### DISCUSSION TITLE
saving types

### DISCUSSION THREAD

**Discussion by eidichap** [mikel-brostrom/yolo_tracking#528] [General]
what are the different ways in which I can save the processed file?

**Comment by mikel-brostrom** [comment]
If you mean in which ways you can save the video file:

https://github.com/mikel-brostrom/Yolov5_StrongSORT_OSNet/blob/6f9f294914ad503a1efc850e8d8dddfb46ef146a/track.py#L267

then you have following codecs:

https://web.archive.org/web/20210506045746/https://www.fourcc.org/codecs.php

The most popular ones are: H.264, H.265, MPEG-4... And here you have a good basic explanation for codecs:

https://www.brid.tv/best-video-codecs-to-use-and-how-to-choose-the-right-one/

**Comment by eidichap** [comment]
how to save the count and its class in a file? i have been doing it but i have not got favorable results.

**Comment by mikel-brostrom** [comment]
There is no counting functionality in this repo. You would need to implement it yourself
